.github/workflows/release-typegen.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
           cd ts/typegen
           make package-typegen
       - name: Set up gcloud
-        uses: GoogleCloudPlatform/github-actions/setup-gcloud@v0.6.0
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@v0
         with:
           version: '388.0.0'
           service_account_email: ${{ secrets.GCP_SA_EMAIL }}
```

Comment:
Seems that the service_account_email is deprecated.  
https://github.com/google-github-actions/setup-gcloud#authentication-inputs

Reply:
I added a TODO comment about the deprecation fields. I'd not change it now, since this action is a release task, it may take multiple rounds to get it right which IMO not worth it right now.
I changed `v0.6.0` to its hash. Hash should work since we are using it in https://github.com/GoogleContainerTools/kpt/blob/a4151b921bc6679b0de1ed4498805750a7fe41a2/.github/workflows/firebase-hosting-merge.yml#L35.